### PR TITLE
VR-57

### LINKED ISSUE #57: Apply a default pagination
You're unable to give your own default pagination. It always is 10, 25 and 100 rows. You should be able to customize the pagination for creating different scenarios.

current-display-of-situation

### PR DESCRIPTION
**Pull request by martn001**
- Updated packages and README file
- Added two new props for controlling pagenation

closes #57

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,6 +173,24 @@ example for crudActionsAfter slot:
...
+ * Multiple row selecting per page
+ *
+ * @return array
+ */
+```
+
+```
+/**
+ * defaultRowsPerPage
+ * The number of how many rows their will been showed at start
```
Comment (koenhoeijmakers) [inline README.md]: The default number of rows.

File: README.md [hunk 2]
```diff
@@ -173,6 +173,24 @@ example for crudActionsAfter slot:
  */
 ```
 
+```
+/**
+ * rowsPerPageItems
+ * Multiple row selecting per page
```
Comment (koenhoeijmakers) [inline README.md]: I'd rename this to `rowsPerPageOptions` and then the description to: "The rows per page option selection"

Comment (ricardovanlaarhoven) [reply README.md]: Keep it the same as vuetify?

Comment (koenhoeijmakers) [reply README.md]: Ah didn't know! Lets keep it that way then :)

**Pull request merged**